pom.xml

```diff
@@ -18,15 +18,14 @@
 		<jdk.version>17</jdk.version>
 		<cds.services.version>2.1.0</cds.services.version>
 		<spring.boot.version>3.1.2</spring.boot.version>
-		<cloud.sdk.version>4.18.0</cloud.sdk.version>
-		<xsuaa.version>3.1.0</xsuaa.version>
+		<xsuaa.version>3.0.1</xsuaa.version>
```

Comment:
Let's remove XSUAA dependency management. It was mainly introduced to ensure consistency with Cloud SDK, but this was also removed from this project.